_posts/2017-12-28-foreman-community-newsletter---december-2017.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2017
+date: 2017-12-28 15:10:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Migrating from mailing lists, Katello 3.5, and the usual round-up
+---
+
+Happy holidays!
+
+### Migrating from Google Groups
+
+Our mailing lists are moving to Discourse on 1st Jan!
+See the [discourse_announcement] for more details.
+
+For those who missed it, the past month there were discussions and
+demos showing Discourse. After a migration poll last month the move to
+Discourse got a lot of support.
+The results were discussed [here](https://youtu.be/g8XnAKfG9xY?t=99). 
+
+### Katello 3.5 released
+
+On the 7th of Dec Katello 3.5 was released RC2, see the [post][k_3_5] for details.
+The release fixed 28 bugs found in RC2 so thank you to all that tested,
```

Comment:
this reads a bit unclear... The two message are
1. Katello 3.5 (final) is released
2. 28 issues were fixed from RC2 to the final release

correct?

Reply:
copy+paste error :) yes both are correct, I'll fix it